src/lib/Cashlink.ts

```diff
@@ -27,22 +27,33 @@ class Cashlink {
 
     set value(value: number) {
         if (this._value && (this._immutable || this.state !== CashlinkState.UNCHARGED)) {
-            throw new Error('Cashlink is immutable');
+            throw new Error('Cannot set value, Cashlink is immutable');
         }
-        if (!Nimiq.NumberUtils.isUint64(value) || value === 0) throw new Error('Malformed value');
+        if (!Nimiq.NumberUtils.isUint64(value) || value === 0) throw new Error('Malformed Cashlink value');
         this._value = value;
     }
 
+    set fee(fee: number) {
+        if (this.state === CashlinkState.CLAIMED) {
+            throw new Error('Cannot set fee, Cashlink is already claimed');
```

Comment:
I think it should not be an error/exception when a fee is set after it is useful, as the fee - different to the value and the message - has no effect on the cashlink url (which is the reason why value and message are immutable).

Reply:
See discussion in https://github.com/nimiq/hub/pull/364#discussion_r366049491